File: codigos_alunos/questao7/java/ErroAtalho.java
package Questao7.java;

public class ErroAtalho {
    public static void main(String[] args) {
        double capacidade_total_gb = 500.0;
        double espaco_ocupado_gb = 420.0;
        
        if ((espaco_ocupado_gb / capacidade_total_gb) > 0.8) {
            System.out.printf("Ocupado: %.0f%%. Alerta Crítico!\n", (espaco_ocupado_gb / capacidade_total_gb) * 100);
        } else {
            System.out.println("Status do armazenamento: Estável");
        }
    }
}

File: codigos_alunos/questao7/python/erro_atalho.py
capacidade_total_gb = 500
espaco_ocupado_gb = 420

if (espaco_ocupado_gb / capacidade_total_gb) > 0.8:
    print(f"O espaço ocupado é {espaco_ocupado_gb / capacidade_total_gb:.0%}. Alerta Crítico!")
else:
    print("Status do armazenamento: Estável")